lambdas/service/app.py

```diff
@@ -1055,6 +1065,7 @@ def get_items_in_cart(cart_id):
         ]
     }
     """
+    cart_id = None if cart_id == 'default' else cart_id
```

Comment:
This change is not in the discussion but I think it would make getting the list of items of the default cart as easy as adding or removing items from the default cart.